KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -86,7 +86,10 @@ The current membership of the committee is (listed alphabetically by first name)
 | Stefano Fioravanzo        | Canonical | [StefanoFioravanzo](https://github.com/StefanoFioravanzo/) |
 | Valentina Rodriguez Sosa  | Red Hat   | [varodrig](https://github.com/varodrig/)                   |        
           
- 
+ ## Lead members
+| Name                | Organization  | GitHub                                              | Role               |
+| ------------------- | ------------- | --------------------------------------------------- | ------------------ |
+| Tarek Abouzeid      | Telia         | [tarekabouzeid](https://github.com/tarekabouzeid/)  | Social Media Lead  |  
```

Comment:
Do we really need to have definition of the role (e.g. Social Media Lead) for KOC leads  ?
As described here: https://github.com/kubeflow/community/blob/ddb0401d1a96696a469108a601296be530bf7990/KUBEFLOW-OUTREACH-COMMITTEE.md#committee-structure, KOC leads can do many things including but not limited to social media activities.

WDYT @varodrig ?